streams/layout/src/main/java/com/hortonworks/streamline/streams/layout/component/rule/expression/Window.java

```diff
@@ -108,7 +108,9 @@ public String toString() {
     private WindowParam windowLength;
     private WindowParam slidingInterval;
     private String tsField;
+    private String[] tsFields;
```

Comment:
have either tsField or tsFields not both. Make sure it works for both Aggregate and Join.

Reply:
Aggregate doesn't (and shouldnt) use tsFields.  
Join doesn't use tsField.  This doesn't break Aggregate.